app/toolbar/toolbar.component.html

```diff
@@ -15,48 +14,34 @@
             {{ lang.display }}
         </span>
     </div>
-  </span>
-  <div class="dropdown toolbar-element notification" *ngIf="loggedIn">
+  </div>
+  <div class="toolbar-element" *ngIf="loggedIn">
     <a [routerLink]="['/notifications/']">
       <span>{{numberOfUnreadNotifications}}</span>
-      <md-icon>chat_bubble_outline</md-icon>
+      <md-icon>message</md-icon>
     </a>
   </div>
-  <div class="dropdown toolbar-element" *ngIf="loggedIn">
-    <a class="dropdown-toggle" type="button" id="dropdownProfile"
-       data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
-      <!-- No picture ?
-      <div class="row-picture">
-        <img class="circle" src="http://lorempixel.com/56/56/people/1" alt="icon"/>
-      </div>
-      -->
-      <span class="profile-name">{{currentUser.email}}</span>
+  <div class="toolbar-element" *ngIf="loggedIn">
+    <a (click)="toggleMenu()">
+      {{ currentUser.email }}
       <md-icon>arrow_drop_down</md-icon>
     </a>
-    <ul class="dropdown-menu" aria-labelledby="dropdownProfile">
+    <ul class="toolbar-menu" *ngIf="loggedIn && menuOpen">
       <li><a [routerLink]="['/notifications']">
-        <h2>
-          <md-icon>message</md-icon>
-          {{'my notifications' | translate}}
-        </h2>
+        <md-icon>message</md-icon>
+        {{ 'my notifications' | translate }}
       </a></li>
       <li><a [routerLink]="['/my-topics']">
-        <h2>
-          <md-icon>library_books</md-icon>
-          {{'my topics' | translate}}
-        </h2>
-      </a></li>
+        <md-icon>library_books</md-icon>
+        {{ 'my topics' | translate }}
+        </a></li>
       <li><a [routerLink]="['/manage-profile']">
-        <h2>
-          <md-icon>person</md-icon>
-          {{'my profile' | translate}}
-        </h2>
+        <md-icon>person</md-icon>
+        {{ 'my profile' | translate }}
       </a></li>
       <li><a (click)="logout()">
-        <h2>
-          <md-icon>logout</md-icon>
-          {{'logout' | translate}}
-        </h2>
+        <md-icon>cancel</md-icon>
```

Comment:
A Materical Icon `logout` doesn't exist, so I chose the one which fits best in my opinion ...